conflation/mapillary.py

```diff
@@ -1,60 +1,106 @@
-from dateutil import parser
-import multiprocessing
+import datetime
+import requests
 import os
 import pickle
-import time
+import multiprocessing
+from dateutil import parser
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
 
-import requests
+from conflation import util
+from conflation import trace_filter
 
-from . import util
+SEQUENCES_PER_PAGE_DEFAULT = 10  # How many sequences to receive on each page of the API call
+IMAGES_PER_PAGE_DEFAULT = 1000  # How many sequences to receive on each page of the API call
+# We only look for sequences beyond the start date, by default a year ago
```

Comment:
nit: might be easier to understand if it were worded around age `by default we only consider sequences up to a year old`